src/sass/style.scss

```diff
@@ -258,6 +258,7 @@ pre .lines .line {
 	display: block;
 	padding-right: 0.33333em;
     color: $color-blue-30;
+	user-select: none;
```

Comment:
I think Safari still wants the -webkit- prefix. But even with prefix it's not working. Only in Chrome. Not a showstopper, but maybe you can double check? I'd also consider applying this to the parent `.lines {}`instead to avoid having the container highlighted when selecting content